$OpenBSD: patch-src_winwidget_h,v 1.1 2013/12/12 09:20:52 dcoppa Exp $

Fix '--borderless' not working on 64bit systems
(upstream git commit b5a719d0dc475066eafc5810c2bdc49bca18713a)

--- src/winwidget.h.orig	Tue Jun 11 08:28:26 2013
+++ src/winwidget.h	Thu Dec 12 09:59:49 2013
@@ -61,11 +61,11 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALI
 
 /* Motif window hints */
 typedef struct _mwmhints {
-	CARD32 flags;
-	CARD32 functions;
-	CARD32 decorations;
-	INT32 input_mode;
-	CARD32 status;
+	unsigned long flags;
+	unsigned long functions;
+	unsigned long decorations;
+	long input_mode;
+	unsigned long status;
 } MWMHints;
 
 enum win_type {
